src/commands/eval.js

```diff
@@ -46,9 +46,9 @@ async function writeFormattedShell(file, str) {
  */
 async function writeFormattedOutput(file, data, format) {
   if (format === "json") {
-    await writeFormattedJson(file, data);
+    return await writeFormattedJson(file, data);
```

Comment:
fix redundant `return await`.

Since the encapsulating fn is `async`, it is okay to return a promise. The only time you need to `return await` is inside a try/catch so that the stack trace comes back to the subject line, rather than where it is caught up the chain.

```suggestion
    return writeFormattedJson(file, data);
```